test/es-module/test-typescript.mjs

```diff
@@ -361,7 +361,7 @@ test('expect process.features.typescript to be \'strip\' when --experimental-str
   ]);
 
   strictEqual(result.stderr, '');
-  strictEqual(result.stdout, 'strip\n');
+  strictEqual(result.stdout, process.config.variables.node_use_amaro ? 'strip\n' : 'false\n');
```

Comment:
1. When Node.js is compiled without type stripping, is `--experimental-strip-types` a no-op? Or does it throw?

2. This test case will never occur, as the test is skipped without Amaro.

Reply:
> 1. When Node.js is compiled without type stripping, is `--experimental-strip-types` a no-op?

It's not a no-op, the flag is "unaffected" by Amaro's presence – but of course, `node` won't be able to strip types, but as long as you're only using non-TS files, you can use the flag even when Amaro is not included IIUC

Reply:
the line 6 skips this whole test file if amaro is not enabled. could this case be moved to a new test file?

Reply:
This test case could also be moved to before the file is skipped